kong/plugins/ldap-auth/access.lua

```diff
@@ -50,17 +50,28 @@ local function ldap_authenticate(given_username, given_password, conf)
 
   sock:settimeout(conf.timeout)
 
-  ok, err = sock:connect(conf.ldap_host, conf.ldap_port)
+  local opts = {}
+
+  -- keep TLS connections in a separate pool to avoid reusing non-secure
+  -- connections and vica versa, because StartTLS use the same port 
```

Comment:
typo: vice-versa

Reply:
@hishamhm good eye, thanks for catching that!